### ISSUE TITLE
CycleCloud shared service does not deploy

### ISSUE THREAD

**Issue by TonyWildish-BH** [microsoft/AzureTRE#4026]
**Describe the bug**
CycleCloud template does not deploy, it fails hard due to a mismatch with command-line arguments in terraform:

```
Error: Value for undeclared variable.
A variable named "arm_client_secret" was assigned on the command line, but the root module does not declare a variable of that name. To use this value, add a "variable" block to the configuration.

Error: Value for undeclared variable.
A variable named "arm_tenant_id" was assigned on the command line, but the root module does not declare a variable of that name. To use this value, add a "variable" block to the configuration.

Error: Value for undeclared variable
A variable named "arm_use_msi" was assigned on the command line, but the root module does not declare a variable of that name. To use this value, add a "variable" block to the configuration.

Error: Value for undeclared variable
A variable named "arm_client_id" was assigned on the command line, but the root module does not declare a variable of that name. To use this value, add a "variable" block to the configuration
```

Is this service deprecated? If so, can it be removed from the code base?

If it's not deprecated, can it be fixed, instead, and documented?

**Steps to reproduce**

1. Deploy the CycleCloud shared service

**Comment by tim-allen-ck** [comment]
Fixes for this can be found in this issue #3933 
PR to be raised to fix this issue.

**Comment by marrobi** [comment]
Closing as fixed in #4050
